tox.ini

```diff
@@ -50,6 +50,15 @@ usedevelop = true
 deps =
     -r{toxinidir}/requirements.txt
 
+# This is used for Windows, since binary name is different
+[testenv:mkvenv-win]
+basepython = python.exe
```

Comment:
Well if they have python27 in their path, they'll have other problems trying to run qutebrowser since QB doesn't support python27. I suppose we could require windows users to modify basepython to reflect the exact version of python they have installed so that tox will always find it, but I found this solution (specifying the exe only) to be the best compromise that will work if users are using a supported python environment without requiring them to edit tox.ini.